docs/conf.py

```diff
@@ -92,7 +101,7 @@
 # -- Options for extensions --------------------------------------------------
 
 # nbsphinx configuration values
-html_sourcelink_suffix = '' # Don't add .txt suffix to source files
+html_sourcelink_suffix = ""  # Don"t add .txt suffix to source files
```

Comment:
```suggestion
html_sourcelink_suffix = ""  # Don't add .txt suffix to source files
```

Reply:
oooops. ;-) I over did the find and replace.